utils/bootstrap-node

```diff
@@ -187,19 +187,33 @@ case $phase in
        ;;
 esac
 
+# It's critical to check for the service state in Consul catalogue.
+# Otherwise, if synchronisation of service status between Consul
+# servers delays (for some reason), hax may get 'offline' state from
+# Consul for already started m0d process (spoiling its ha_link).
+wait_m0mkfs_done()
+{
```

Comment:
[nit,style] We usually put `{` [on the same line](https://github.com/bahamas10/bash-style-guide#functions) with function name.
```sh
wait_m0mkfs_done() {
```

Reply:
Right, my fault. It's been a few months I didn't write in bash and here you are - I forgot how to do it. :)

Will fix.